jsonschema/schema.go

```diff
@@ -105,15 +105,15 @@ func SchemaFromFile(schemaPath string, oneOfType string) (*Schema, error) {
 }
 
 // SchemaTypes will parse the given file and report which top level allOfTypes and oneOfTypes are found in that schema.
```

Comment:
Update the comment to let users of this know what the 3rd return value is.

Reply:
Thanks!